modules/gridmap/doc_classes/GridMapEditorPlugin.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="GridMapEditorPlugin" inherits="EditorPlugin" keywords="tilemap" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../doc/class.xsd">
+	<brief_description>
+		Editor for [GridMap] nodes.
+	</brief_description>
+	<description>
+		GridMapEditorPlugin provides access to the [GridMap] editor functionality.
+	</description>
+	<tutorials>
+	</tutorials>
+	<methods>
+		<method name="clear_selection">
+			<return type="void" />
+			<description>
+				Deselect any currently selected cells.
+			</description>
+		</method>
+		<method name="get_current_grid_map" qualifiers="const">
+			<return type="GridMap" />
+			<description>
+				Returns the [GridMap] node currently edited by the grid map editor.
+			</description>
+		</method>
+		<method name="get_selected_cells" qualifiers="const">
+			<return type="Array" />
+			<description>
+				Returns an array of [Vector3i]s with the selected cells' coordinates.
+			</description>
+		</method>
+		<method name="get_selected_palette_item" qualifiers="const">
+			<return type="int" />
+			<description>
+				Returns the index of the selected [MeshLibrary] item in the grid map editor's palette or [code]-1[/code] if no item is selected.
+				[b]Note:[/b] The indices may not be in the same order as they appear in the editor's interface.
```

Comment:
```suggestion
				[b]Note:[/b] The indices might not be in the same order as they appear in the editor's interface.
```
Possibly this. Current phrasing is correct but ambiguous. "May not" can also mean that the indices are *not allowed* to be in the same order, rather than there being *no guarantee* of the order.

Reply:
Fixed it.